app/src/main/java/fr/neamar/kiss/preference/ColorPreference.java

```diff
@@ -65,6 +70,16 @@ public void onColorSelected(int color) {
         this.getDialog().dismiss();
     }
 
+    private void selectButton(Button button) {
+        Context context = getContext();
+        TypedValue tv = new TypedValue();
+        boolean found = context.getTheme().resolveAttribute(android.R.attr.textColor, tv, true);
```

Comment:
Out of curiosity, was this ever false in your tests?

Reply:
No and it never should be, but its good to have i believe.